### PR TITLE
avoid repetitive errors on adding fdb entry if it already exists

### PR DESCRIPTION
**Pull request by rajatchopra**
cc @openshift/networking

### CODE REVIEW

File: pkg/router/f5/f5.go [hunk 1]
```diff
@@ -977,7 +977,12 @@ func (f5 *f5LTM) AddVtep(ipStr string) error {
 		Name:     macAddr,
 		Endpoint: ipStr,
 	}
-	return f5.post(url, payload, nil)
+	err = f5.post(url, payload, nil)
+	if err != nil && err.(F5Error).httpStatusCode != HTTP_CONFLICT_CODE {
```
Comment (rajatchopra) [inline pkg/router/f5/f5.go]: We are constructing this error type.
https://github.com/rajatchopra/origin/blob/f5_fdb/pkg/router/f5/f5.go#L407
The field httpStatusCode is always filled up either with nil value '0' if the request failed without obtaining the http status code or with the actual status code here:
https://github.com/rajatchopra/origin/blob/f5_fdb/pkg/router/f5/f5.go#L430

**Comment by rajatchopra** [comment]
Example error that this code tries to bypass:
```
E1104 17:41:07.977028       1 plugin.go:521] Error in adding node '10.3.89.134' to F5s FDB - Encountered an error on
POST request to URL https://10.3.88.54/mgmt/tm/net/fdb/tunnel/~Common~vxlan5000/records: HTTP code: 409; error from F5:
01020066:3: The requested L2 forward tunnel (/Common/vxlan5000 0a:0a:0a:03:59:86) already exists in partition Common.
```

File: pkg/router/f5/f5.go (hunk 1 above)
Comment (dcbw) [inline pkg/router/f5/f5.go]: Ok, cool.

**Comment by dcbw** [comment]
LGTM

**Pull request merged**